sor/src/main/java/com/bazaarvoice/emodb/sor/core/DefaultDataStore.java

```diff
@@ -483,6 +469,9 @@ public boolean isChangeDeltaRedundant(UUID changeId) {
                                         @Nullable UUID start, @Nullable UUID end, boolean reversed, long limit, ReadConsistency consistency) {
         checkLegalTableName(tableName);
         checkNotNull(key, "key");
+        if (includeAuditInformation) {
+            throw new UnsupportedOperationException("Audit data is no longer accessable via EmoDB app servers.");
```

Comment:
Suggestion:  Maybe create a custom exception such as `AuditsUnavailableException` extends `RuntimeException`.  This doesn't change anything, but when returning an error code via the REST API catching a known exception like this can be used to return a 400 (bad request) instead of a 500 (server error).

Reply:
I actually created an execption mapper for UnsupportedOperationException, so it already returns a 400. I would be happy to create a custom exception for it though.